data.sql

```diff
@@ -21,4 +21,43 @@ VALUES ('Charmander', '2020-02-08', 0, False, -11),
         ('Blossom', '1998-10-13', 3, True, 17),
         ('Ditto', '2022-05-14', 4, True, 22);
 
+-- Insert into Owners table
+
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34),
+        ('Jennifer Orwell', 19),
+        ('Bob', 45),
+        ('Melody Pond', 77),
+        ('Dean Winchester', 14),
+        ('Jodie Whittaker', 38);
+
+-- Insert into Species table
+
+INSERT INTO species (name)
+VALUES ('Pokemon'),
+        ('Digimon');
+
+-- Modify animals table
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon')
+WHERE name LIKE '%mon';
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon')
+WHERE species_id IS NULL;
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith')
+WHERE name = 'Agumon';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
+WHERE name = 'Gabumon' OR name = 'Pikachu';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob')
+WHERE name = 'Devimon' OR name = 'Plantmon';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond')
+WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
+WHERE name = 'Angemon' OR name = 'Boarmon';
```

Comment:
Please add these commands ni your schema.sql so that everything that creates or changes the table's structure is saved in the schema.sql file ✔

Reply:
Hello! Thank you for your suggestions but I believe I was following the project requirements. [Project Requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Insert%20the%20following,data.sql.)